go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/giantswarm/releases
 go 1.13
 
 require (
+	github.com/Masterminds/semver v1.5.0
```

Comment:
Are we certain that this is the semver lib to use? It feels like we use different ones everywhere :(

Reply:
This is in a lot of other repos :man_shrugging:

Reply:
This should be `github.com/Masterminds/semver/v3 v3.1.0` since v1 and v2 aren't under active development and aren't (fully) go module compatible AFAIK. See https://github.com/Masterminds/semver#package-versions We should upgrade this or notify owners if we see it in other repos too.

Reply:
Ah yeah, good catch. I always forget I have to version the import statements themselves